src/PolykeyClient.ts

```diff
@@ -114,13 +108,13 @@ class PolykeyClient {
 
   public async stop() {
     this.logger.info(`Stopping ${this.constructor.name}`);
-    await this.grpcClient.destroy();
     await this.session.stop();
     this.logger.info(`Stopped ${this.constructor.name}`);
   }
 
   public async destroy() {
     this.logger.info(`Destroying ${this.constructor.name}`);
+    await this.rpcClient.destroy();
```

Comment:
So we moved from stopping to destroying for the `RPCClient`?

Reply:
Yeah, the `RPCClient` doesn't really have any state anymore. It's just a container for some data and an interface for making calls. It doesn't really need to be destroyed even. it only prevents it from being used from then on.